### PR TITLE
Updated Readme.md and fix the issue #328

### PR DESCRIPTION
**Pull request by shiv4nsh**
Updated the link of the akka.io website in the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # AKKA.IO
 
-This repository contains the source of the [akka.io](akka.io) website. It's built using Jekyll, and designed with a touch of Twitter's Bootstrap, and other homebaked ingredients.
+This repository contains the source of the [akka.io](http:/akka.io) website. It's built using Jekyll, and designed with a touch of Twitter's Bootstrap, and other homebaked ingredients.
```
Comment (ktoso) [inline README.md]: Thanks, nice catch. Please `http://` though :)

**Comment by shiv4nsh** [comment]
@ktoso : So can we merge this.? :)

**Comment by ktoso** [comment]
> @ktoso : So can we merge this.? :)

Please address the above comment before we merge, thanks

**Comment by ktoso** [comment]
Well, or I'll just fix it. Thanks for the catch!

**Pull request merged**